models/cow_protocol/cow_protocol_trades.sql

```diff
@@ -0,0 +1,38 @@
+{{ config(
+        alias ='trades'
+        )
+}}
+SELECT *
+FROM
+(
+        SELECT
+            'ethereum' AS blockchain,
+            'CoW Protocol' AS project,
+            '1' AS version,
+            TRY_CAST(date_trunc('DAY', block_time) AS date) AS block_date,
+            block_time,
+            buy_token AS token_bought_symbol,
+            sell_token AS token_sold_symbol,
+            token_pair,
+            units_bought AS token_bought_amount,
+            units_sold AS token_sold_amount,
+            atoms_bought AS token_bought_amount_raw,
+            atoms_sold AS token_sold_amount_raw,
+            usd_value AS amount_usd,
+            buy_token_address AS token_bought_address,
+            sell_token_address AS token_sold_address,
+            trader AS taker,
+            '' AS maker,
```

Comment:
@bh2smith Need some of your input in which is which on these two, as I'm not really sure. I've been reference the Uniswap implementation so far.

Reply:
If I remember correctly, for Uniswap, the AMM is considered the "maker" (aka market maker). In our protocol there is no maker. Might be worth looking at what 1inch considers the "maker" for comparison. 

Maybe we want to leave this NULL instead of an empty string?

Also, @josojo might have something to add here.

Reply:
```suggestion
            trader AS taker,
            NULL AS maker,
```